make_functions.sh

```diff
@@ -13,7 +13,7 @@ JUJU_DB_VERSION=${JUJU_DB_VERSION:-}
 
 # Docker variables
 OCI_BUILDER=${OCI_BUILDER:-docker}
-DOCKER_USERNAME=${DOCKER_USERNAME:-docker.io/jujusolutions}
+GHCR_USERNAME=${GHCR_USERNAME:-ghcr.io/juju}
```

Comment:
Can we make this generic, eg OCI_REGISTRY_USERNAME
in our jenkins CI we have `DOCKER_USERNAME` and so will need to change that as well.

Reply:
that sounds better, will change it thanks ian!